Update json-filtering.mdx (#2056)
hashicorp/terraform-website: content/cloud-docs/workspaces/json-filtering.mdx

## Entering a Filter

> Filters are entered by putting the filter in the apply named **filter** box in

Reviewer: Ah! thank you for catching this. I think actually we can make it even shorter. How about this?

```suggestion
Enter a filter into the **filter** box in
```
